### DISCUSSION TITLE
screenshare with camera using adaptive streaming

### DISCUSSION THREAD

**Discussion by nocpbxchap** [ant-media/Ant-Media-Server#5053] [General]
Hi, i am having issues when using screen share witch camera and adaptive streaming

without any transcoding the stream works perfect no delay but when using adaptive streaming seems like low fps issue to give an example.

i know these is because we are using to stream , screen share and camera so the aspect ratio may conflict, but these only happen with adaptive streaming.

Any configuration suggest as i know these issue .

**Comment by lastpeony** [comment]
Hi
Adaptive bitrate streaming requires a significant amount of resources, and insufficient server resources can result in performance issues. To help diagnose any issues, could you kindly provide your server logs, CPU load, and streaming details, including publish parameters if you are using software such as OBS? This information will be useful in identifying and addressing any potential resource limitations.
